ckanext/subscribe/fanstatic/subscribe.css

```diff
@@ -13,7 +13,16 @@
     margin-bottom: 5px;
 }
 #subscribe-form input {
-    width: 90px;
+    display: block;
+    margin-bottom: 5px;
+    padding: 6px 6px 6px 6px;
+    width: auto;
```

Comment:
When width is set to auto the input field is not the same size as the submit button for me.
Should it be set to 175px?